package.json

```diff
@@ -32,65 +32,59 @@
   "scripts": {
     "start": "node scripts/build.js --bundle --serve",
     "build": "node scripts/build.js --bundle --types --copydir \"docs/dist\"",
-    "prepublishOnly": "npm run build && npm run test",
+    "release": "npm run spellcheck && npm run prettier:check && npm run ts-check && npm run build && npm run test",
```

Comment:
This doesn't actually trigger a release (releases are performed manually atm), so the name is a bit confusing. Is there a better name or pattern to follow here, like `verify` or something?

Reply:
Yeah I'm not sure, maybe "prepare for release" is what I was thinking. You are right that this doesn't make sense. Maybe `release-build` or something since this is building and validating everything before a release? Not sure a great name though.